.gitignore

```diff
@@ -174,7 +174,6 @@ ruby/tests/generated_code_proto2_pb.rb
 ruby/tests/multi_level_nesting_test_pb.rb
 ruby/tests/test_import_proto2_pb.rb
 ruby/tests/test_ruby_package_proto2_pb.rb
-ruby/Gemfile.lock
```

Comment:
I think of this as a safety mechanism to prevent accidentally committing `Gemfile.lock`. Is there a reason why we have to remove this?

Reply:
Nope, I can add this back.  I think an earlier iteration *did* check it in